lib/index.js

```diff
@@ -192,27 +153,13 @@ export default {
           return [];
         }
 
-        // Construct the list of linters to be passed to the CLI by looking at
-        // the linters made available for our language grammar by the LinterMap,
-        // and whether or not the engines are enabled in a user's config file.
-        if (Object.prototype.hasOwnProperty.call(linterMap, grammarName) === true) {
-          // for (const linter of linterMap[grammarName]) {
-          linterMap[grammarName].forEach((linter) => {
-            linterNames.add(linter);
-          });
-        }
-        const configEnabledEngines = getEnabledEngines(configurationFilePath);
-        const linterEnabledEngines = Array.from(linterNames).filter(
-          linterName => configEnabledEngines.has(linterName));
-
         // Construct the command line invocation which runs the Code Climate CLI
         const relpath = atom.project.relativizePath(filePath).pop();
-        const execArgs = ['analyze', '-f', 'json'].concat(
-          getEngineArgs(linterEnabledEngines),
-          [relpath, '</dev/null']);
+        const execArgs = ['analyze', '-f', 'json', relpath];
         const execOpts = {
           cwd: dirname(configurationFilePath),
           uniqueKey: `linter-codeclimate::${relpath}`,
+          env: process.env,
```

Comment:
I think I know why it fails with the newer CC CLI.

The primary change in that new CLI can scan piped in source but it needs a filename to determine what engines to run against it. For obvious reasons there can be only one file name specified in that case.

It appears CLI improperly detects this case and errors out because it thinks there are two paths specified.

It thinks there are two paths because Node passes `</dev/null` as a literal parameter to the CLI. It doesn't do redirection as a shell would do. But even if it did I don't see why it's needed here.

In any case, as I mentioned before, this PR removes `</dev/null` bit so the command should not fail because of this specific reason.

@cgalvarez What happens if you directly run `codeclimate analyze -f json -e fixme -e rubocop cool_code.rb` in the terminal?